assets/js/src/core/components/grid/columns/types/language-select/language-cell.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import { type DefaultCellProps } from '@Pimcore/components/grid/columns/default-cell'
+import React from 'react'
+import { SelectCell } from '@Pimcore/components/grid/columns/types/select/select-cell'
+import {
+  useGetSystemSettingsQuery
+} from '@Pimcore/modules/asset/editor/shared-tab-manager/tabs/custom-metadata/settings-slice.gen'
+
+export const LanguageCell = (props: DefaultCellProps): React.JSX.Element => {
+  const { data, isLoading } = useGetSystemSettingsQuery()
```

Comment:
hmm, I'm just thinking if it would be more efficient to centralize the settings, somehow. For example in the app loader. And we just then use a simple use hook to access all the data. I think this would give us a benefit compared to individual integrations in the different components. 

What do you think? :)

Reply:
i guess that makes sense - i could create a store and then we fetch it from there ...
i would do that in the followup :)